tox.ini

```diff
@@ -11,7 +11,7 @@ python =
   pypy: pypypy
 
 [testenv]
-usedevelop = false
+usedevelop = True
```

Comment:
This change is risky, if something makes the tests depend on development code, what guarantees that the distributed code then works?

Opened https://github.com/learningequality/iceqube/issues/42

Reply:
It didn't work without it, and it's what we use in Kolibri.

Would be good to know what we should do to make our tests run against production packages.

Reply:
https://github.com/learningequality/kolibri/issues/3310 :wink: